assignments/assignment-8/assignment-2.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8</title>
+    </head>
+    <body>
+        <form>
+            <label for="fullNameInput">Your Full Name</label>
+            <input type="text" name="fullName" id="fullNameInput" />
+
+            <label for="emailInput">Your Email</label>
+            <input type="email" name="email" id="emailInput" />
+
+            <br />
+
+            <label for="ratingInput">Rate your Confidence Level</label>
+            <input
+                type="range"
+                name="rating"
+                id="ratingInput"
+                min="1"
+                max="10"
+            />
+
+            <label for="ageInput">Your Age</label>
+            <input type="number" name="age" id="ageInput" min="18" max="99" />
+
+            <p>Select your Favourite Quiz Category</p>
+            <label for="historyCategoryCheckbox">History</label>
+            <input
+                type="checkbox"
+                name="isHistoryChecked"
+                id="historyCategoryCheckbox"
+            />
+            <br />
+            <label for="scienceCategoryCheckbox">Science</label>
+            <input
+                type="checkbox"
+                name="isScienceChecked"
+                id="scienceCategoryCheckbox"
+            />
+            <br />
+            <label for="sportCategoryCheckbox">Sport</label>
+            <input
+                type="checkbox"
+                name="isSportChecked"
+                id="sportCategoryCheckbox"
+            />
+            <br />
+            <label for="literatureCategoryCheckbox">Literature</label>
+            <input
+                type="checkbox"
+                name="isLiteratureChecked"
+                id="literatureCategoryCheckbox"
+            />
+
+            <p>Select Preferred Quiz Format</p>
+            <label for="multipleChoiceOption">Multiple Choice</label>
```

Comment:
all grouped radio inputs need to have the same name, in this case preferredQuizFormat. And give the value to each radio: 'Multiple Choice',  'Fill in the Blank', 'True/False'